### ISSUE TITLE
[4.0] Cannot insert an article in a custom module with wysiwyg editor

### ISSUE THREAD

**Issue by PhilETaylor** [joomla/joomla-cms#20496]
### Steps to reproduce the issue

Attempt to insert a ARTICLE using the CMS Content dropdown in the editor in admin when editing a **CUSTOM MODULE** 

Select the article from the list in the modal

### Expected result

No error, clicking allows modal to close and insert article in content

### Actual result

JS error, modal stays open 

`Uncaught TypeError: Cannot read property 'checked' of null`

![screen recording 2018-05-19 at 10 22 pm](https://user-images.githubusercontent.com/400092/40273154-1f4db1de-5bb3-11e8-969f-46aca2ff2cb6.gif)

### System information (as much as possible)

79fd949

### Additional comments

similar to https://github.com/joomla/joomla-cms/issues/20495 BUT DIFFERENT ENOUGH to raise different issue as different error message and probable cause

**Comment by brianteeman** [comment]
Confirmed for article as shown in the video - I assume the mention of contacts in the steps to reproduce is a typo?

**Comment by PhilETaylor** [comment]
correct

issue for CONTACTS is here https://github.com/joomla/joomla-cms/issues/20495  different issue

**Comment by brianteeman** [comment]
also confirmed for inserting an article in an article

**Comment by PhilETaylor** [comment]
also get this when double clicking the ROW of a contact in the contacts modal

**Comment by infograf768** [comment]
Tested here inserting article in article or in Custom Module:
The code for the article IS inserted but the modal does not close automatically.
We have also an error:
```
TypeError: i is null[Learn More]
multiselect.min.js:1:1246
<anonymous>
http://localhost:8888/newfolder/joomla40/media/system/js/multiselect.min.js:1:1246
```

**Comment by dgrammatiko** [comment]
> but the modal does not close automatically

That is a known issue, we're (js group) preparing a solution for this: https://github.com/joomla/joomla-cms/pull/19830

Please be patient till we get that API right

**Comment by Quy** [comment]
Same JS error as #20492 and fixed in PR #20659 except for the auto closing of the modal.

**Comment by brianteeman** [comment]
I can confirm the js is fixed by #20659 and as the modal close is being addressed in #19830 I am going to close this - thanks
